sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/DatabaseCrudTest.java

```diff
@@ -108,6 +114,25 @@ public void deleteDatabase_DoesntExist() throws Exception {
         validateFailure(deleteObservable, validator);
     }
 
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
+    public void readReplaceAutoscaleThroughput() throws Exception {
+        safeDeleteDatabase(client.getDatabase("newTestDatabase"));
```

Comment:
tests should not use static database name to avoid any potential issue with runnings tests in parallel.

Please use CosmosDatabaseForTest.generateId()

Reply:
Makes sense. Changed